exercises/grade-school/grade_school_test.rb

```diff
@@ -4,85 +4,104 @@
 require_relative 'grade_school'
 
 class SchoolTest < Minitest::Test
-  attr_reader :school
-
-  def setup
-    @school = School.new
-  end
-
-  def test_an_empty_school
-    assert_equal({}, school.to_h)
+  def test_empty_grade
+    subject = School.new
```

Comment:
I have to disagree with this. The most straightforward way to represent an ordered list in Ruby is with an array. Using strings with delimiters is more work and less useful to the consumer of the query.

Plus, a hash is being used later for `students_by_grade`. I definitely don't want to try to consume that as a string instead of a hash. And if it uses a ruby data structure, why not this?